docs/docs/noir/standard_library/meta/expr.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Expr
+---
+
+`std::meta::expr` contains methods on the built-in `Expr` type for quoted, syntactically valid expressions.
+
+## Methods
+
+### as_array
+
+#include_code as_array noir_stdlib/src/meta/expr.nr rust
+
+If this expression is an array, this returns a slice of each element in the array.
```

Comment:
We return a slice even when calling `as_array`?

Reply:
Because this function works on array literals of any size, and we don't know the size of these literals from just the `Expr` type so we have to return a slice of each field.